pybossa/cache/helpers.py

```diff
@@ -30,33 +30,22 @@
 session = db.slave_session
 
 
-def n_available_tasks(project_id, user_id=None, user_ip=None):
+def n_available_tasks(project_id, user_id=None):
     """Return the number of tasks for a given project a user can contribute to.
```

Comment:
From the changes, it seems this function would return number of incomplete tasks. 
1. Is there impact or rephrasing function name to something like "n_incomplete_tasks" that matches w/ what it is doing? Also update docstring
2. the user_id arg passed as input can be removed.
3. n_available_tasks is still being called from places where it seems to collect stats as it was originally that would need to be changed as well. Example [here](https://github.com/bloomberg/pybossa/blob/project-info-bug/pybossa/api/__init__.py#L317) and [here](https://github.com/bloomberg/pybossa/blob/project-info-bug/pybossa/view/projects.py#L787)

Reply:
Not sure whether we want to keep the old `n_available_tasks` function. 
[this](https://github.com/bloomberg/pybossa/blob/d9242d8fe03831130b2a7052b6c994255809f744/pybossa/view/projects.py#L787) is the change pointed out by the ticket. And [here](https://github.com/bloomberg/pybossa/blob/d9242d8fe03831130b2a7052b6c994255809f744/pybossa/api/__init__.py#L295) looks like it calls the function for "# remaining tasks" too. We may want to keep consistency in these places otherwise it could be confusing if users see different result on different pages for the same field "number of tasks remaining".